### PR TITLE
feat: create constants for employment types

### PR DESCRIPTION
**Pull request by Nebulis**

### CODE REVIEW

File: packages/constants/src/mcf.test.ts [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+      {id: 8, description: 'Senior Executive'},
+      {id: 9, description: 'Executive'},
+      {id: 10, description: 'Junior Executive'},
+      {id: 11, description: 'Non-executive'},
+      {id: 12, description: 'Fresh/entry level'},
+    ]);
+  });
+  it('should have correct job categories', () => {
+    expect(JOB_CATEGORIES).to.deep.equal([
+      {jobCategoryId: 1, jobCategoryName: 'Accounting / Auditing / Taxation'},
```
Comment (eijiew) [inline packages/constants/src/mcf.test.ts]: so we decide to have `jobCategoryId`?

File: packages/constants/src/msf.ts [hunk 2]
```diff
@@ -0,0 +1,273 @@
+export const EMPLOYMENT_TYPES = [
+  {id: '02', name: 'Temporary'},
+  {id: '03', name: 'Contract'},
+  {id: '05', name: 'Part Time'},
+  {id: '06', name: 'Freelance'},
+  {id: '07', name: 'Permanent'},
+  {id: '08', name: 'Full Time'},
+  {id: '09', name: 'Flexi Time'},
```
Comment (lizzieloh) [inline packages/constants/src/msf.ts]: I have checked MSF is actually returning as `Flexi Work` but our `api-profile` somehow put it as `Flexi Time` ^^"

File: packages/constants/src/mcf.test.ts (hunk 1 above)
Comment (Nebulis) [reply packages/constants/src/mcf.test.ts]: for now yeah, we can make a final decision next sprint

Comment (eijiew) [reply packages/constants/src/mcf.test.ts]: ok

**Comment by zephinzer** [comment]
Trusting @eijiew and @lizzieloh!

**Pull request merged**